tests/api_tests.py

```diff
@@ -141,6 +141,20 @@ def test_get_all_cameras(self):
         result = self.app.get(route)
         self.assertEquals(result.status_code, 200)
 
+    def test_login(self):
+        data = {"username" : "johndoe",
+                "password": "passWord"}
+        result = self.app.post(
+            "/users/login", data=json.dumps(data), content_type="application/json")
+        self.assertEqual(result.status_code, 200)
+
+    def test_create_login(self):
+        data = {"username" : "johndoe",
+                "password": "passWord"}
```

Comment:
@rgoluch  lmaoooo

Reply:
thank, we love to see it @katebrune 
it truly is necessary and the only way to name a passWord variable